UIViewController-DisplayInDrawer/Classes/UIViewController+PresentDrawerController.swift

```diff
@@ -7,6 +7,14 @@ public protocol DrawerConfiguration: class {
     ///If you return nil, there will only be two positions, top and bottom
     func middlePositionY(for parentHeight: CGFloat) -> CGFloat?
     func bottomPositionY(for parentHeight: CGFloat) -> CGFloat
+    /**
+     Background color of the drawer controller. If value is nil, then light effect is applied to the background.
+     */
+    var drawerBackgroundColor: UIColor? { get }
+    /**
+     Corner radius of the drawer controller.
+     */
+    var drawerCornerRadius: CGFloat { get }
```

Comment:
Make default implementation for these new methods, to preserve current behaviour for legacy apps